config/defaults.mk

```diff
@@ -301,45 +300,24 @@ PUMI_OPT = -I$(PUMI_DIR)/include
 PUMI_LIB = -L$(PUMI_DIR)/lib -lpumi -lcrv -lma -lmds -lapf -lpcu -lgmi -lparma\
    -llion -lmth -lapf_zoltan -lspr
 
-# CUDA library configuration
-ifeq ($(MFEM_USE_CUDA),YES)
-   ifndef CUDA_DIR
-      CUDA_DIR := /usr/local/cuda
-   endif
-   CUDA_OPT := -I$(CUDA_DIR)/include
-   CUDA_LIB := -L$(CUDA_DIR)/lib64 -lcuda -lcudart
-endif
-
-# MPI library configuration
-ifeq ($(MFEM_USE_MPI),YES)
```

Comment:
This is needed for the parallel build with CUDA, no?

Reply:
It turns out, you can just tell `nvcc` to use the mpi wrapper and that seems to work. Still need to test that on other systems though.

Reply:
> you can just tell nvcc to use the mpi wrapper and that seems to work

That will be ideal, if it works on LC, Linux and Mac? Should we document it in `INSTALL`?